### PR TITLE
Add badge for Integration, Chat and PR

### PR DESCRIPTION
**Pull request by jayeshathila**
Signed-off-by: jayeshathila <sharma.jayesh52@gmail.com>

### Description
Add badges for Integration tests, Chat and PR welcome
 
 
### Check List
- [x] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by vamshin — APPROVED** [review]
LGTM! Thanks

File: README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+[![Integration](https://github.com/opensearch-project/opensearch-ruby/actions/workflows/integration.yml/badge.svg)](https://github.com/opensearch-project/opensearch-py/actions/workflows/main.yml)
```
Comment (VijayanB) [inline README.md:1]:
```suggestion

[![Integration](https://github.com/opensearch-project/opensearch-ruby/actions/workflows/main.yml/badge.svg)](https://github.com/opensearch-project/opensearch-ruby/actions/workflows/main.yml)
```

Comment (jayeshathila) [reply README.md:1]: Fixed

**Comment by jayeshathila** [comment]
Raised another PR from forked repo.

**Pull request closed without merging**